Zilon.Core/Zilon.Core.Tests/MapGenerators/RoomStyle/RoomMapFactoryTests.cs

```diff
@@ -57,10 +60,12 @@ public void Create_RealRoomRandom_NoExceptions(int diceSeed)
             var factory = new RoomMapFactory(roomGenerator);
             var sectorScheme = CreateSectorScheme();
 
+            var sectorFactoryOptions = new SectorMapFactoryOptions(sectorScheme.MapGeneratorOptions);
+
             // ACT
             Func<Task> act = async () =>
             {
-                var map = await factory.CreateAsync(sectorScheme);
+                var map = await factory.CreateAsync(sectorFactoryOptions).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'map'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)